package.json

```diff
@@ -14,9 +14,13 @@
     "grunt-sass": "^1.0.0",
     "grunt-contrib-less": "1.3.0",
     "grunt-less-to-sass": "0.0.10",
-    "jit-grunt": "^0.9.1"
+    "jit-grunt": "^0.9.1",
+    "swagger-ui": "^2.2.5"
   },
   "devDependencies": {
     "grunt-contrib-watch": "~0.6.1"
+  },
+  "scripts": {
+    "postinstall": "rm -rf public/swagger-ui; cp -r node_modules/swagger-ui/dist public/swagger-ui; sed -i .orig \"s/defaultModelRendering: 'schema'/defaultModelRendering: 'model'/\" public/swagger-ui/index.html; sed -i .orig 's/url = \".*\"/url = \"..\\/..\\/api\\/v1\\?swagger\"/' public/swagger-ui/index.html"
```

Comment:
You should be able to install the swagger-ui easily without packaging the whole thing. Needing grunt for that is quite heavy, but if you think it's better, I could add a separate grunt task and make it part of the package task.